### PR TITLE
code/__HELPERS/pronouns.dm.

### PR DESCRIPTION
**Pull request by Ghommie**
- [x] Create pronouns.dm.
- [x] Change the names of several related inventory and equipment defines and procs to match /tg/'s.
- [x] Replace the macros with pronouns procs.
- [x] triple check for typos or skipped macro. Probably.

:cl: Ghommie
code: replaced chat message text macros with pronouns procs.
/:cl:

### CODE REVIEW

File: code/__DEFINES/equipment.dm [hunk 1]
```diff
@@ -49,18 +49,18 @@
 //flags_inv_hide
 //Bit flags for the flags_inv_hide variable, which determine when a piece of clothing hides another. IE a helmet hiding glasses.
 
-#define HIDEGLOVES		1
```
Comment (spookydonut) [inline code/__DEFINES/equipment.dm]: This has nothing to do with code/__HELPERS/pronouns.dm

File: code/datums/diseases/mob_procs.dm [hunk 2]
```diff
@@ -142,8 +142,8 @@
 			if(isobj(wear_suit))
 				Cl = wear_suit
 				. = prob((Cl.permeability_coefficient*100) - 1)
-			if(. && isobj(WEAR_BODY))
```
Comment (spookydonut) [inline code/datums/diseases/mob_procs.dm]: why have you changed this? this has nothing to do with pronouns, split this off into a separate PR please

File: code/datums/mind.dm [hunk 3]
```diff
@@ -343,7 +343,7 @@ datum/mind
 			switch(href_list["common"])
 				if("undress")
 					for(var/obj/item/W in current)
-						current.drop_inv_item_on_ground(W)
+						current.dropItemToGround(W)
```
Comment (spookydonut) [inline code/datums/mind.dm]: this is not in your list of changes, why are you changing the names of inventory procs?

Comment (Ghommie) [reply code/datums/mind.dm]: actually this is in the list of changes.
It's to save up brain matter from having to memorize different names for the same procs by the by.

File: code/__DEFINES/equipment.dm [hunk 4]
```diff
@@ -60,8 +60,7 @@
 #define HIDETOPHAIR		256		// temporarily removes the user's hair overlay. Leaves facial hair.
 #define HIDEALLHAIR		512		// temporarily removes the user's hair, facial and otherwise.
 #define HIDETAIL 		1024
-#define HIDEFACE		2056	//Dictates whether we appear as unknown.
-
+#define HIDEFACE	2056	//Dictates whether we appear as unknown.
```
Comment (spookydonut) [inline code/__DEFINES/equipment.dm]: needless formatting :P

File: code/game/gamemodes/colonialmarines/huntergames.dm [hunk 5]
```diff
@@ -161,17 +161,17 @@ var/waiting_for_drop_votes = 0
...
-	for(var/obj/item/ammo_magazine/M in item_list) 
+	for(var/obj/item/ammo_magazine/M in item_list)
 		qdel(M)
 
 	for(var/mob/new_player/player in player_list)
 		if(player && player.ready)
 			if(!player.mind && player.client)
 				player.mind = new /datum/mind(player.key)
 			player.mind.assigned_role = "ROLE"
-
```
Comment (spookydonut) [inline code/game/gamemodes/colonialmarines/huntergames.dm]: grrr

Comment (spookydonut) [reply code/game/gamemodes/colonialmarines/huntergames.dm]: no actual code changes to this file

File: code/game/jobs/job/job.dm [hunk 6]
```diff
@@ -126,24 +126,24 @@
...
-	if(available_in_days(C) == 0) 
+	if(available_in_days(C) == 0)
 		return TRUE	//If available in 0 days, the player is old enough to play. Can be compared to null, but I think this is clearer.
 
 /datum/job/proc/available_in_days(client/C)
 	//Checking the player's age is only possible through a db connection, so if there isn't one, player age will be a text string instead.
-	if(!istype(C) || !config.use_age_restriction_for_jobs || !isnum(C.player_age) || !isnum(minimal_player_age)) 
+	if(!istype(C) || !config.use_age_restriction_for_jobs || !isnum(C.player_age) || !isnum(minimal_player_age))
 		return FALSE //One of the few times when returning 0 is the proper behavior.
 	return max(0, minimal_player_age - C.player_age)
```
Comment (spookydonut) [inline code/game/jobs/job/job.dm]: no code changes just formatting in this file

File: code/game/jobs/job/marines.dm [hunk 7]
```diff
@@ -18,7 +18,7 @@
 
 /datum/job/marine/generate_entry_conditions(mob/living/carbon/human/H)
 	. = ..()
-	if(flags_startup_parameters & ROLE_ADD_TO_MODE) 
+	if(flags_startup_parameters & ROLE_ADD_TO_MODE)
```
Comment (spookydonut) [inline code/game/jobs/job/marines.dm]: same here

File: code/game/jobs/job/mercenaries.dm [hunk 8]
```diff
@@ -7,7 +7,7 @@
 	access = ALL_MARINE_ACCESS
 	minimal_access = ALL_MARINE_ACCESS
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT
-	
+
```
Comment (spookydonut) [inline code/game/jobs/job/mercenaries.dm]: same here

File: code/game/jobs/job/shipside.dm [hunk 9]
```diff
@@ -49,7 +49,7 @@ Godspeed, commander! And remember, you are not above the law."}
 /datum/job/command/commander/announce_entry_message(mob/living/carbon/human/H)
 	. = ..()
 	sleep(15)
-	if(H?.loc && flags_startup_parameters & ROLE_ADD_TO_MODE) 
+	if(H?.loc && flags_startup_parameters & ROLE_ADD_TO_MODE)
```
Comment (spookydonut) [inline code/game/jobs/job/shipside.dm]: same here

File: code/game/objects/items/cards_ids.dm [hunk 10]
```diff
@@ -352,7 +352,7 @@
 					msg += "\"[fallen_names[x]] - [fallen_assignements[x]]\""
 				else
 					msg += "\"[fallen_names[x]] - [fallen_assignements[x]]\", "
-			 
+
```
Comment (spookydonut) [inline code/game/objects/items/cards_ids.dm]: same here

File: code/game/objects/items/devices/defibrillator.dm [hunk 11]
```diff
@@ -20,7 +20,7 @@
 	origin_tech = "biotech=3"
 
 	suicide_act(mob/user)
```
Comment (spookydonut) [inline code/game/objects/items/devices/defibrillator.dm]: pathing

File: code/game/objects/items/legcuffs.dm [hunk 12]
```diff
@@ -20,7 +20,7 @@
 	var/armed = 0
 
 	suicide_act(mob/user)
```
Comment (spookydonut) [inline code/game/objects/items/legcuffs.dm]: pathing

File: code/game/objects/items/stacks/cable_coil.dm [hunk 13]
```diff
@@ -20,9 +20,9 @@
 	attack_verb = list("whipped", "lashed", "disciplined", "flogged")
 	stack_id = "cable coil"
 
-	suicide_act(mob/user)
-		viewers(user) << "<span class='warning'><b>[user] is strangling \himself with the [src.name]! It looks like \he's trying to commit suicide.</b></span>"
-		return(OXYLOSS)
+/obj/item/stack/cable_coil/suicide_act(mob/user)
+	user.visible_message("<span class='warning'><b>[user] is strangling [p_them()]self with the [src.name]! It looks like [user.p_theyre()] trying to commit suicide.</b></span>")
```
Comment (spookydonut) [inline code/game/objects/items/stacks/cable_coil.dm]: class='danger'

File: code/game/objects/items/storage/belt.dm [hunk 14]
```diff
@@ -320,7 +320,7 @@
 		if(M.current_rounds)
 			if(contents.len < storage_slots)
 				to_chat(user, "<span class='notice'>You start refilling [src] with [M].</span>")
-				if(!do_after(user, 15, TRUE, 5, BUSY_ICON_GENERIC))
```
Comment (spookydonut) [inline code/game/objects/items/storage/belt.dm]: same here

File: code/game/objects/items/storage/pouch.dm [hunk 15]
```diff
@@ -353,7 +353,7 @@
 		if(M.contents.len)
 			if(contents.len < storage_slots)
 				to_chat(user, "<span class='notice'>You start refilling [src] with [M].</span>")
-				if(!do_after(user, 15, TRUE, 5, BUSY_ICON_GENERIC))
```
Comment (spookydonut) [inline code/game/objects/items/storage/pouch.dm]: same here

File: code/game/objects/items/weapons/blades.dm [hunk 16]
```diff
@@ -12,9 +12,9 @@
 	w_class = 3
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 
-	suicide_act(mob/user)
-		viewers(user) << "\red <b>[user] is falling on the [src.name]! It looks like \he's trying to commit suicide.</b>"
-		return(BRUTELOSS)
+/obj/item/weapon/claymore/suicide_act(mob/user)
+	viewers(user) << "\red <b>[user] is falling on the [src.name]! It looks like [user.p_theyre()] trying to commit suicide.</b>"
```
Comment (spookydonut) [inline code/game/objects/items/weapons/blades.dm]: fix this text macro, replace with danger

File: code/game/objects/structures/kitchen_spike.dm [hunk 17]
```diff
@@ -25,7 +25,7 @@
 				occupied = 1
 				meat = 5
 				meattype = 1
-				visible_message("\red [user] has forced [M] onto the spike, killing them instantly!")
+				visible_message("\red [user] has forced [M] onto the spike, killing [M.p_them()] instantly!")
```
Comment (spookydonut) [inline code/game/objects/structures/kitchen_spike.dm]: fix the macro

File: code/modules/admin/verbs/randomverbs.dm [hunk 18]
```diff
@@ -48,7 +48,7 @@
 	set category = null
 	set name = "Subtle Message"
 
-	if(!ismob(M))	
+	if(!ismob(M))
```
Comment (spookydonut) [inline code/modules/admin/verbs/randomverbs.dm]: same here

File: code/modules/client/client defines.dm [hunk 19]
```diff
@@ -1,4 +1,8 @@
 /client
+		//////////////////////
+		//BLACK MAGIC THINGS//
+		//////////////////////
+	parent_type = /datum
```
Comment (spookydonut) [inline code/modules/client/client defines.dm]: no no no no no this is from a different PR.

File: code/modules/cm_marines/equipment/sentries.dm [hunk 20]
```diff
@@ -79,7 +79,7 @@
 		S.update_icon()
 		qdel(src)
 
-	
+
```
Comment (spookydonut) [inline code/modules/cm_marines/equipment/sentries.dm]: same here

File: code/modules/cm_marines/overwatch.dm [hunk 21]
```diff
@@ -1254,7 +1254,7 @@
 		dat += "<b><font color=red>NONE!</font></b><br>"
 	dat += get_squad_info_ending()
 	return dat
-	
+
```
Comment (spookydonut) [inline code/modules/cm_marines/overwatch.dm]: same here

File: code/modules/mob/inventory.dm [hunk 22]
```diff
@@ -139,26 +139,41 @@
...
-	if(hasvar(src,"wear_suit")) if(src:wear_suit) items += src:wear_suit
-//	if(hasvar(src,"w_radio")) if(src:w_radio) items += src:w_radio  commenting this out since headsets go on your ears now PLEASE DON'T BE MAD KEELIN
-	if(hasvar(src,"w_uniform")) if(src:w_uniform) items += src:w_uniform
-
-	//if(hasvar(src,"l_hand")) if(src:l_hand) items += src:l_hand
-	//if(hasvar(src,"r_hand")) if(src:r_hand) items += src:r_hand
-
-	return items
+//Daily reminder to murder this - Remie from /tg/station
+/mob/living/proc/get_equipped_items(include_pockets = FALSE)
```
Comment (spookydonut) [inline code/modules/mob/inventory.dm]: this is from one of your inventory PRs

File: code/modules/mob/living/carbon/human/emote.dm [hunk 23]
```diff
@@ -184,7 +184,7 @@
 					if (M.canmove && !M.r_hand && !M.is_mob_restrained())
 						message = "<B>[comm_paygrade][src]</B> shakes hands with [M]."
 					else
-						message = "<B>[comm_paygrade][src]</B> holds out \his hand to [M]."
+						message = "<B>[comm_paygrade][src]</B> holds out [p_their()] hand to [M]."
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/human/emote.dm]: replace the <b> while you're at it

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 24]
```diff
@@ -187,8 +187,34 @@
 		s_store = null
 		update_inv_s_store()
 
-
-
+/mob/living/carbon/human/get_equipped_items(include_pockets = FALSE)
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/human/inventory.dm]: from inventory PR

File: code/modules/mob/living/carbon/inventory.dm [hunk 25]
```diff
@@ -34,6 +34,13 @@
 		legcuffed = null
 		legcuff_update()
 
+/mob/living/carbon/get_equipped_items(include_pockets = FALSE)
+	var/list/items = list()
+	if(back)
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/inventory.dm]: inventory stuff

File: code/modules/mob/living/carbon/xenomorph/Abilities.dm [hunk 26]
```diff
@@ -1022,7 +1022,7 @@
 	if(!T.xeno_caste.deevolves_to)
 		to_chat(X, "<span class='xenowarning'>[T] can't be deevolved.</span>")
 		return
-	
+
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/Abilities.dm]: same here

File: code/modules/mob/living/carbon/xenomorph/Castes/Carrier.dm [hunk 27]
```diff
@@ -150,14 +150,14 @@
...
-	// *** Pheromones *** //	
+	// *** Pheromones *** //
 	aura_strength = 2.5
 
 	// *** Carrier Abilities *** //
 	huggers_max = 11
 	hugger_delay = 1.0 SECONDS
-	eggs_max = 6	
-					
+	eggs_max = 6
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/Castes/Carrier.dm]: same here

File: code/modules/mob/living/carbon/xenomorph/Evolution.dm [hunk 28]
```diff
@@ -77,7 +77,7 @@
 
 	if(!new_caste_type)
 		to_chat(src, "EVO8: Something went wrong with evolving")
-		return 
+		return
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/Evolution.dm]: same here

File: code/modules/projectiles/updated_projectiles/ammunition.dm [hunk 29]
```diff
@@ -456,9 +456,9 @@ Turn() or Shift() as there is virtually no overhead. ~N
 
 
 /obj/item/ammo_magazine/shotgunbox/update_icon()
-	if(current_rounds > 0) 
+	if(current_rounds > 0)
 		icon_state = "[base_icon_state]_deployed"
-	else 
+	else
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/ammunition.dm]: same here

File: code/modules/projectiles/updated_projectiles/gun_attachables.dm [hunk 30]
```diff
@@ -1140,12 +1140,12 @@ Defined in conflicts.dm of the #defines folder.
...
 			return O
-	
-	T = get_step(T, user.dir) 
+
+	T = get_step(T, user.dir)
 	for(var/obj/O in T)
 		if((istype(O, /obj/structure/window_frame)))
 			return O
-	
+
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/gun_attachables.dm]: same here

File: code/modules/projectiles/updated_projectiles/gun_helpers.dm [hunk 31]
```diff
@@ -129,14 +129,14 @@ DEFINES in setup.dm, referenced here.
...
-	else 
+	else
 		return ..()
 
 
 /obj/item/weapon/gun/throw_at(atom/target, range, speed, thrower)
 	if( harness_check(thrower) )
 		to_chat(usr, "<span class='warning'>\The [src] clanks on the ground.</span>")
-	else 
+	else
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/gun_helpers.dm]: same here

File: code/modules/projectiles/updated_projectiles/gun_system.dm [hunk 32]
```diff
@@ -287,13 +287,13 @@
 	A.remove_hud(user)
 
 	return TRUE
-	
+
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/gun_system.dm]: same here

File: code/modules/projectiles/updated_projectiles/guns/misc.dm [hunk 33]
```diff
@@ -214,7 +214,7 @@
 
 /obj/item/weapon/gun/launcher/spike/delete_bullet(obj/item/projectile/projectile_to_fire, refund = 0)
 	qdel(projectile_to_fire)
-	if(refund) 
+	if(refund)
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/guns/misc.dm]: same here

File: code/game/objects/items/stacks/cable_coil.dm (hunk 13 above)
Comment (Ghommie) [reply code/game/objects/items/stacks/cable_coil.dm]: All suicide act messages are class='warning' here afaik. Should be just class='rose' since it's drama at most.

**Comment by Rohesie** [comment]
Also conflicts.

**Comment by Rohesie** [comment]
Look clear to me, waiting on @spookydonut to finish it.
